test/backend/routes.spec.js

```diff
@@ -10,23 +10,25 @@ import {
 import {getSetting, saveSetting} from '../../backend/settings.js';
 import fs from 'fs';
 import {
-    username,
     accessToken,
+    apacheDrillConnections,
+    apacheDrillStorage,
     apiKey,
-    validFid,
-    validUids,
-    createGrid,
     configuration,
-    sqlConnections,
-    mysqlConnection,
+    createGrid,
     elasticsearchConnections,
+    fakeCerts,
+    mysqlConnection,
     publicReadableS3Connections,
-    apacheDrillConnections,
-    apacheDrillStorage,
+    sqlConnections,
     testConnections,
-    testSqlConnections
+    testSqlConnections,
+    username,
+    validFid,
+    validUids
 } from './utils.js';
 
+const access_token = accessToken;
```

Comment:
Yes it's used in some of those oauth calls since the key should be `access_token` so writing `{access_token}` is easier... Wanna use `{access_token: accessToken}` ?